docs/server/getSnippets.js

```diff
@@ -0,0 +1,30 @@
+/* global __dirname */
+import fs from 'mz/fs';
```

Comment:
I'm full of questions 😄. Do we still need `mz` now that `util.promisify` gets us essentially the same thing?

Reply:
Good point. Done.

Reply:
I actually killed off all this special snippet API stuff anyway and just use `raw-loader` to load in snippets. TLDR: for a static site, Next will still try to run getInitialProps on the client, but there's no server, so of course it fails.